examples/nlp_finetuning/start_without_hpo.py

```diff
@@ -0,0 +1,47 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import boto3
+from syne_tune.backend.sagemaker_backend.sagemaker_utils import get_execution_role
+
+from renate.training import run_training_job
+
+config_space = {
+    "optimizer": "SGD",
+    "momentum": 0.9,
+    "weight_decay": 0.0,
+    "learning_rate": 0.001,
+    "alpha": 0.5,
+    "batch_size": 32,
+    "memory_batch_size": 32,
+    "memory_size": 300,
+    "loss_normalization": 0,
+    "loss_weight": 0.5,
+}
+
+if __name__ == "__main__":
+    AWS_ID = boto3.client("sts").get_caller_identity().get("Account")
+    AWS_REGION = "us-west-2"  # use your AWS preferred region here
+
+    run_training_job(
```

Comment:
do we ever use `chunk_id=1`? I like that you've used two datasets here. Why not use it as part of the example?

Reply:
None of the examples actually perform more than one update, right? We would need to wait for the first run to finish, before starting the second one.

Reply:
true, in this case we are using sagemaker. if we want to keep it that way, my question is why the config file has the option to use a second dataset.